arrows/klv/klv_timeline.h

```diff
@@ -0,0 +1,102 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// \brief Declaration of KLV timeline class.
+
+#include <arrows/klv/klv_packet.h>
+#include <arrows/klv/klv_set.h>
+
+#include <vital/util/interval_map.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace klv {
+
+// ----------------------------------------------------------------------------
+/// Tracks the value of KLV fields over time.
+class KWIVER_ALGO_KLV_EXPORT klv_timeline
```

Comment:
There is such a nice writeup of this class as part of this PR. Would you consider putting that text here also as permanent documentation since it provides high level information?